bb-bootstrap-alerts-module/includes/frontend.php

```diff
@@ -11,12 +11,12 @@
 
 	<!-- Navigation Link -->
 	<?php if($settings->bbn_dropdown_link=='yes'): ?>
-		<a href="<?php echo ($nav_link=$settings->bbn_navigation_link != '')? $nav_link=$settings->bbn_navigation_link : '#' ;?>" target="<?php echo $nav_target=$settings->bbn_navigation_target;?>" >
+		<a href="<?php echo esc_url($nav_link=$settings->bbn_navigation_link != '')? $nav_link=$settings->bbn_navigation_link : '#' ;?>" target="<?php echo esc_attr( $nav_target=$settings->bbn_navigation_target );?>" >
 	<?php endif; ?>
 			<!-- Alert class -->
-			<div class="alert <?php echo $settings->bbn_dropdown_field; ?> fade in" >
+			<div class="alert <?php echo esc_attr( $settings->bbn_dropdown_field ); ?> fade in" >
 				<!-- Close Button -->
-				<?php if( $settings->bbn_closable == 'yes' ): ?>
+				<?php if( esc_attr( $settings->bbn_closable == 'yes' ) ): ?>
```

Comment:
I think  $settings->bbn_closable only  should go through esc_attr()

why whole if condition is added inside it?

Reply:
Yes, correct. Will update
sanitizing `esc_attr` only on `$settings->bbn_closable` to make sure its value is safe for HTML attributes and ensure security.